templates/config.edn

```diff
@@ -294,56 +330,53 @@
  ;; - {time}: capture time
  ;; - {date}: capture date using current date format, use `[[{date}]]` to get a page reference
  ;; - {text}: text that users selected before sharing.
- ;; - {url}: url or assets path for media files stored in Logseq.
- ;; You can also reorder them, or even only use one or two of them in the template.
- ;; You can also insert or format any text in the template as shown in the following examples.
+ ;; - {url}: URL or assets path for media files stored in Logseq.
+ ;; You can also reorder them or use only one or two of them in the template.
+ ;; You can also insert or format any text in the template, as shown in the following examples.
  ;; :quick-capture-templates
  ;; {:text "[[quick capture]] **{time}**: {text} from {url}"
  ;;  :media "[[quick capture]] **{time}**: {url}"}
 
- ;; Quick capture options
+ ;; Quick capture options.
  ;; :quick-capture-options {:insert-today? false :redirect-page? false :default-page "my page"}
 
  ;; File sync options
  ;; Ignore these files when syncing, regexp is supported.
- ;; :file-sync/ignore-files []
-
- ;; dwim (do what I mean) for Enter key when editing.
- ;; Context-awareness of Enter key makes editing more easily
- ; :dwim/settings {
- ;   :admonition&src?  true
- ;   :markup?          false
- ;   :block-ref?       true
- ;   :page-ref?        true
- ;   :properties?      true
- ;   :list?            true
- ; }
-
- ;; Decide the way to escape the special characters in the page title.
+ :file-sync/ignore-files []
+
+ ;; Configure the Enter key behavior for
+ ;; context-aware editing with DWIM (Do What I Mean).
+ ;; :dwim/settings {
+ ;;   :admonition&src?  true
+ ;;   :markup?          false
+ ;;   :block-ref?       true
+ ;;   :page-ref?        true
+ ;;   :properties?      true
+ ;;   :list?            true
+ ;; }
+
+ ;; Configure the escaping method for special characters in page titles.
  ;; Warning:
- ;;   This is a dangerous operation. If you want to change the setting,
- ;;   should access the setting `Filename format` and follow the instructions.
- ;;   Or you have to rename all the affected files manually then re-index on all
- ;;   clients after the files are synced. Wrong handling may cause page titles
- ;;   containing special characters to be messy.
- ;; Available values:
+ ;;   This is a dangerous operation. To modify the setting,
+ ;;   access the 'Filename format' setting and follow the instructions.
+ ;;   Othwerwise, You may need to manually rename all affected files and
+ ;;   re-index them on all clients after synchronization.
+ ;;   Incorrect handling may result in messy page titles.
+ ;; Available options:
  ;;   :file/name-format :triple-lowbar
  ;;     ;use triple underscore `___` for slash `/` in page title
  ;;     ;use Percent-encoding for other invalid characters
- :file/name-format :triple-lowbar
-
- ;; specify the format of the filename for journal files
- ;; :journal/file-name-format "yyyy_MM_dd"
```

Comment:
Why was this moved so much higher towards the top? I think given that it's a "dangerous setting" like the one above this, putting it at the bottom makes sense. Things at the bottom are less likely to be interacted with, and signal that to the user.